### PR TITLE
Adding new template for Empty Edge node setup

### PR DESCRIPTION
**Pull request by ezubatov**
This is an empty node setup template
We are using dummy script which just simply does echo to console

### CODE REVIEW

File: EmptyNode/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "clusterName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the existing HDInsight cluster"
+            }
+        },
+        "galleryPackageIdentity": {
```
Comment (travismc1) [inline EmptyNode/azuredeploy.json]: Let's just make this a variable as opposed to a parameter. We had it as a parameter when we were trying to have examples for ISV's to onboard

File: EmptyNode/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                "description": "Number of instances"
+            },
+            "defaultValue": 1
+        },
+        "edgeNodeSetupScriptUri": {
+            "type": "string",
+            "metadata": {
+                "description": "Edgenode setup script"
+            },
+            "defaultValue": "https://raw.githubusercontent.com/ezubatov/Iaas-Applications/master/EmptyNode/scripts/EmptyNodeSetup.sh"
```
Comment (travismc1) [inline EmptyNode/azuredeploy.json]: Be sure to update this after doing the PR.

File: EmptyNode/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+            ]
+        },
+        "edgeNodePrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Edgenode prefix"
+            },
+            "defaultValue": "new"
+        },
+        "instanceCount": {
```
Comment (travismc1) [inline EmptyNode/azuredeploy.json]: We have to set this as a variable until we see what the cost is of changing this in our workflows. There is a strict validation that you can only have one instance. We'll have to dig in further to determine whether that was scoping for testing or if there's other implications in our code that could fail if instanceCount > 1

Comment (travismc1) [inline EmptyNode/azuredeploy.json]: Or we could just set the instanceCount to 1 directly in the json below.

File: EmptyNode/readme.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Add a new empty edge node the the cluster
+
+This template helps to add a new empty edgenode to the cluster -<br>
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fezubatov%2FIaas-Applications%2Fmaster%2FEmptyNode%2Fazuredeploy.json" target="_blank">
```
Comment (travismc1) [inline EmptyNode/readme.md]: Also update this after PR.

**Comment by ezubatov** [comment]
Travis, I have addressed your comments.

**Comment by travismc1** [comment]
Looks good. Feel free to merge

**Pull request merged**